templates/default/sysvinit.service.erb

```diff
@@ -64,7 +64,7 @@ _start() {
     daemon \
          --pidfile=$pidfile \
          --user=$user \
-         " { $exec <%= @daemon_options %> &>> $logfile & } ; echo \$! >| $pidfile "
+         " { $exec <%= @daemon_options %> > $logfile 2>&1 & } ; echo \$! >| $pidfile "
```

Comment:
Whoopsie.  The reason for changing the syntax at all was to fix a test kitchen failure on CentOS 5. See the accompanying [git comment](https://github.com/johnbellone/consul-cookbook/pull/325/commits/49dbde591594ae6c158cb871db862ff060c89476) for more context.

The `>` instead of `>>` was totally accidental. Will fix momentarily and push.